apps/voting/contracts/Voting.sol

```diff
@@ -464,31 +482,33 @@ contract Voting is IForwarder, AragonApp {
     * @return True if the given vote can be executed, false otherwise
     */
     function _canExecute(Vote storage vote_) internal view returns (bool) {
+        // If vote is already executed, it cannot be executed again
         if (vote_.executed) {
             return false;
         }
 
-        // Voting is already decided
-        if (_isValuePct(vote_.yea, vote_.votingPower, vote_.supportRequiredPct)) {
+        // If the vote is already decided and early execution is allowed, it can be executed
+        if (!earlyExecutionDisallowed && _isValuePct(vote_.yea, vote_.votingPower, vote_.supportRequiredPct)) {
             return true;
         }
 
-        // Vote ended?
+        // If the vote is still open but early execution is not allowed, it cannot be executed
```

Comment:
I think this comment is not accurate. It should be something like:
```
If the vote is still open but early execution is not allowed or vote is not decided yet (checked just above), it cannot be executed
```
Or just leave:
```
If the vote is still open, it cannot be executed
```
and assume reader has gone through previous lines, which is probably more consistent with the rest of comments in this functions, that don't include information about previous checks.